packages/functional-tests/src/tests/physics-stack-test.ts

```diff
@@ -7,12 +7,12 @@ import * as MRE from '@microsoft/mixed-reality-extension-sdk';
 
 import { App } from '../app';
 import { Test } from '../test';
-import { Vector3 } from '@microsoft/mixed-reality-extension-sdk';
+import { Vector3, Actor } from '@microsoft/mixed-reality-extension-sdk';
```

Comment:
These are already imported through the above MRE import.  Use MRE.Actor and MRE.Vector3 and remove this added import.